apps/i18n/common/en_us.json

```diff
@@ -750,7 +750,7 @@
   "emptyFunctionBlocksErrorMsg": "The function block needs to have other blocks inside it to work.",
   "emptyFunctionalBlock": "You have a block with an unfilled input.",
   "emptySection": "Your section doesn't have any students! Go to the [Manage Students]({url}) tab to add some.",
-  "emptySurveyOverviewTable": "Because this survey is anonymous, we can only show aggregated results once there are at least 5 submissions.",
+  "emptySurveyOverviewTable": "Because this survey is anonymous, we can only show aggregated results once at least 5 students have submitted their survey and answered the first question of the survey. \n\nIf you have given this survey to your class and are still not seeing responses here, please give the survey to your class again and encourage your students to answer the first question of the survey before they submit the survey. [Learn more about surveys and assessments.](https://support.code.org/hc/en-us/articles/115001331951-Using-lock-settings-for-assessments-and-surveys)",
```

Comment:
This isn't in the ticket, but I'm wondering if we should remove the `en-us` from the link (change to: https://support.code.org/hc/articles/115001331951-Using-lock-settings-for-assessments-and-surveys), it looks like this link still works and the language gets automatically detected.

Reply:
ooo yes good point! Will do